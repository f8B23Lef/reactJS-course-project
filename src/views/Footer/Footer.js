import React from 'react';
import Logo from '../../components/Logo/Logo';
import './Footer.scss';

export default function Footer() {
  return (
    <div className='footer'>
      <Logo />
    </div>
  );
}
