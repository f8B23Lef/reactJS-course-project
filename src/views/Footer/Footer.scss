@import '../../styles/index.scss';

.footer {
  display: flex;
  justify-content: center;
  padding: 20px;
  position: sticky;
  bottom: 0;
  width: 100vw;
  background-color: $grey;
}