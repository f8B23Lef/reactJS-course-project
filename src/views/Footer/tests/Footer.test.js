import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Footer from '../Footer';

jest.mock('../../../components/Logo/Logo', () => () => 'LogoComponent');

describe('Footer view', () => {
  it('renders Footer view snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    );

    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
