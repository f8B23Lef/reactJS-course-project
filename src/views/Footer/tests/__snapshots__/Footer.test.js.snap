// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer view renders Footer view snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    LogoComponent
  </div>
</DocumentFragment>
`;
